### ISSUE TITLE
No Email Request in Comment Box - Safari Only

### ISSUE THREAD

**Issue by davidnwright** [cogdog/truwriter#4]
![screen shot 2016-02-23 at 10 36 25 pm](https://cloud.githubusercontent.com/assets/529132/13277477/fdf98296-da7d-11e5-90d4-fe67f6633e03.png)

In Safari, sometimes, the Email / Name / Website line does not appear. I assume this is because Wordpress assumes user is logged in (as "writer"). With Remove Dashboard Access plugin installed, the "edit" button (see pic) is not an issue, but I usually need to know the name of the commenter so I can give them credit for commenting and to generally keep track that a human is commenting. 

I have not been able to reproduce *every* time. But has happened consistently for some...

At yr leisure!

**Comment by cogdog** [comment]
Yes this is the case, they are still logged in as "writer"-- a long running vexing issue. I tried modifying the automatic timeout period (when wordpress automatically logs them out) but (a) it does not work, and (b) I don't want to make it so it happens while not writing.

Other approaches are putting a button that forces a logout on the confirmation page.  I will take a look at doing that, maybe a giant modal overlay when they are done editing.

I did make a small indicator- if you look in the footer, if there is a (1) that means they are logged in as writer- if they click the "(1)" it will log them out

![splot in](https://cloud.githubusercontent.com/assets/463038/13286144/96a4b662-dad5-11e5-90cf-74c6ddf2132b.jpg)

This was more for a case with some users who had accounts on same multiuser WP for other blogs. But until I fix it, you might have to let students click that link to clear their session.

Will look into a better approach soon.
